beacon-chain/node/p2p_config.go

```diff
@@ -36,9 +37,10 @@ func configureP2P(ctx *cli.Context) (*p2p.Server, error) {
 	if err != nil {
 		return nil, err
 	}
+	metricAdapter := metric.New()
 
-	// TODO(437): Define default adapters for logging, monitoring, etc.
-	adapters := []p2p.Adapter{traceAdapter}
+	// TODO: Define default adapters for logging.
```

Comment:
We need to have `// TODO(#ISSUE_NUM)` to track an issue corresponding to the todo. This is enforced by our CI so it will fail unless we use that format